### PR TITLE
add support for self-hosting

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: src/transforms/ampDisableCacheTransform.js [hunk 1]
```diff
@@ -14,25 +14,89 @@
...
+    }
+  }
+
   const ampOptimizer = AmpOptimizer.create({
-    transformations: ['RemoveAmpAttribute'],
+    transformations: [DownloadRuntime, 'RemoveAmpAttribute', 'RewriteAmpUrls'],
   });
   eleventyConfig.addTransform('amp-disable-cache', async (content, outputPath) => {
-    if (!outputPath.endsWith('.html')) {
+    if (!options.isAmp(outputPath)) {
```
Comment (patrickkettner) [inline src/transforms/ampDisableCacheTransform.js:92]: Are you confident this is always defined? Why are we setting a [default param](https://github.com/ampproject/eleventy-plugin-amp/pull/32/files?short_path=915e09f#diff-0770dc4fd3493744c50d191bc447b2fbR31)?

Comment (sebastianbenz) [reply src/transforms/ampDisableCacheTransform.js:92]: Did you implement this in AmpConfig?

Comment (sebastianbenz) [reply src/transforms/ampDisableCacheTransform.js:92]: I think the default param is legacy and we should remove it.

Comment (sebastianbenz) [reply src/transforms/ampDisableCacheTransform.js:92]: Removed the default param. Good catch!

**Pull request merged**